modules/react/button/lib/TertiaryButton.tsx

```diff
@@ -99,27 +108,27 @@ const getTertiaryButtonColors = (
   } else {
     return {
       default: {
-        icon: children ? themePrimary.main : colors.blackPepper400,
+        icon: hasThemeStyles ? themePrimary.main : colors.blackPepper400,
```

Comment:
if they add this prop with our default theme won't it change the icon color?

Reply:
This is kind of tricky because for tertiary buttons we change the icon color completely and there isn't a 1:1 color match in our default theme